models/reports/sql/generic/location-bookings-line-list.sql

```diff
@@ -0,0 +1,48 @@
+select
+    display_id as "{{ translate_string('general.localisedField.displayId.label', 'Patient ID') }}",
+    first_name as "{{ translate_string('general.localisedField.firstName.label', 'First name') }}",
+    last_name as "{{ translate_string('general.localisedField.lastName.label', 'Last name') }}",
+    to_char(date_of_birth, '{{ var("date_format") }}') as "{{ translate_string('general.localisedField.dateOfBirth.label', 'Date of birth') }}",
+    age as "{{ translate_string('general.localisedField.Age', 'Age') }}",
```

Comment:
StringId does not exist in `https://translations.ops.tamanu.io/v2.28.0.csv`
```suggestion
    age as "{{ translate_string('general.table.column.age', 'Age') }}",
```